tools/editor/multi_node_edit.cpp

```diff
@@ -52,8 +52,16 @@ bool MultiNodeEdit::_set(const StringName& p_name, const Variant& p_value){
 		Node*n=es->get_node(E->get());
 		if (!n)
 			continue;
+		else if (p_value.get_type() == Variant::NODE_PATH) {
+			Node *tonode = n->get_node(p_value);
+			if (tonode) {
+				NodePath p_path = n->get_path_to(tonode);
+				ur->add_do_property(n,name,p_path);
+			}
```

Comment:
M, no, I don't think so, because this would end up with that `/root/EditNode/@@4/@@6/etc` thing. If `tonode == NULL` for some reason then it should be an error, I mean, it's something that shouldn't happen.